### PR TITLE
[mod_articles_category] Showon

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue # .

### Summary of Changes

This adds `showon` attribute to form fields in Articles - Category module.

### Testing Instructions

Code review. Or create Articles - Category module and try different options. Check that fields are shown/hidden correctly.

### Documentation Changes Required
Option from Dynamic Mode Options moved to Module tab. Update help page? https://docs.joomla.org/Help38:Extensions_Module_Manager_Articles_Category

### CODE REVIEW

File: language/en-GB/en-GB.mod_articles_category.ini [hunk 1]
```diff
@@ -52,7 +53,7 @@ MOD_ARTICLES_CATEGORY_FIELD_MODE_DESC="Please select the mode you would like to
 MOD_ARTICLES_CATEGORY_FIELD_MODE_LABEL="Mode"
 MOD_ARTICLES_CATEGORY_FIELD_MONTHYEARFORMAT_DESC="Please enter in a valid date format. See: http://php.net/date for formatting information."
 MOD_ARTICLES_CATEGORY_FIELD_MONTHYEARFORMAT_LABEL="Month and Year Display Format"
-MOD_ARTICLES_CATEGORY_FIELD_RELATIVEDATE_DESC="If Relative Date is selected above, please enter in a numeric day value. Results will be retrieved relative to the current date and the value you enter."
+MOD_ARTICLES_CATEGORY_FIELD_RELATIVEDATE_DESC="Please enter in a numeric day value. Results will be retrieved relative to the current date and the value you enter."
```
Comment (brianteeman) [inline language/en-GB/en-GB.mod_articles_category.ini]: ==> "Please enter a numeric value"

File: modules/mod_articles_category/mod_articles_category.xml [hunk 2]
```diff
@@ -344,24 +344,27 @@
...
 					label="MOD_ARTICLES_CATEGORY_FIELD_ARTICLEGROUPINGDIR_LABEL"
 					description="MOD_ARTICLES_CATEGORY_FIELD_ARTICLEGROUPINGDIR_DESC"
 					default="ksort"
+					showon="article_grouping!:none"
 					>
 					<option value="krsort">MOD_ARTICLES_CATEGORY_OPTION_DESCENDING_VALUE</option>
 					<option value="ksort">MOD_ARTICLES_CATEGORY_OPTION_ASCENDING_VALUE</option>
 				</field>
 
-				<field
```
Comment (brianteeman) [inline modules/mod_articles_category/mod_articles_category.xml:358]: Why is this field removed?

**Review by brianteeman — APPROVED** [review]
approve language string changes

File: modules/mod_articles_category/mod_articles_category.xml (hunk 2 above)
Comment (SharkyKZ) [reply modules/mod_articles_category/mod_articles_category.xml:358]: It's moved above.

Comment (brianteeman) [reply modules/mod_articles_category/mod_articles_category.xml:358]: I need more coffee ;)

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 35e8db5f2cf0d7f0ff20fba0699a2aa0be0f2d1d<br /><br />Test using "Modules: Articles - Category", works as expected.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20950">issues.joomla.org/tracker/joomla-cms/20950</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 35e8db5f2cf0d7f0ff20fba0699a2aa0be0f2d1d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20950">issues.joomla.org/tracker/joomla-cms/20950</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20950">issues.joomla.org/tracker/joomla-cms/20950</a>.</sub>

**Pull request merged**